params/version.go

```diff
@@ -33,12 +33,14 @@ var Version = func() string {
 }()
 
 // VersionWithMeta holds the textual version string including the metadata.
-var VersionWithMeta = func() string {
+var VersionWithMeta, Unstable = func() (string, bool) {
 	v := Version
+	unstable := false
 	if VersionMeta != "" {
 		v += "-" + VersionMeta
+		unstable = true
```

Comment:
This is not true, `VersionMeta` will either be `"stable"` or `"unstable"`, but generally not empty.